crates/runtime/src/dataconnector/mod.rs

```diff
@@ -599,6 +600,14 @@ pub async fn get_data(
         df = df.filter(filter).map_err(find_datafusion_root)?;
     }
 
+    if tracing::enabled!(Level::DEBUG) {
+        if let Ok(explained) = df.clone().explain(false, false) {
+            if let Ok(explained) = explained.to_string().await {
+                tracing::debug!("get_data df.explain for {}: \n{}", table_name, explained);
```

Comment:
Can we have this more developer friendly?
get_data df.explain for
->
Data refresh plan for  / Base data refresh plan for..